x-pack/plugin/esql/qa/testFixtures/src/main/resources/qstr-function.csv-spec

```diff
@@ -0,0 +1,114 @@
+###############################################
+# Tests for QSTR function
+#
+
+qstrWithField
+required_capability: qstr_function
+
+// tag::qstr-with-field[]
+from books 
+| where qstr("author: Faulkner")
+| keep book_no, author 
+| sort book_no 
+| limit 5;
+// end::qstr-with-field[]
+
+book_no:keyword | author:text
```

Comment:
A result tag(qstr-with-field-result) is missing, the tag is referenced in `docs/reference/esql/functions/examples/qstr.asciidoc`

Reply:
I see, I added that in 70dd5d2a9610463857c4188d75c03a730c85f71c